locales/en.json

```diff
@@ -837,6 +837,9 @@
     "holdings": "Holdings",
     "gains": "Gains",
     "showQr": "Show QR",
+    "searchNoResultsTitle": "No Assets On Selected Chain",
+    "searchNoResultsText": "Explore The Artist's Protfolio On Other Networks ({0})",
+    "searchNoResultsButton": "Explore On {0}",
```

Comment:
why duplicated key?

Reply:
They are in different scopes and they are used in different places. 
`general. searchNoResultsTitle ` and `profile.searchNoResultsTitle`

Reply:
I don't understand, there is no point of having duplicate text in translation file even if "different" scopes

Reply:
Oh never mind. I forget to delete the similar key in this file : ) my bad